content/en/docs/instrumentation/ruby/getting-started.md

```diff
@@ -1,58 +1,120 @@
 ---
 title: Getting Started
+description: Get telemetry from your app in less than 5 minutes!
 aliases: [/docs/instrumentation/ruby/getting_started]
+spelling: cSpell:ignore truffleruby sinatra rolldice struct darwin Tracestate
+spelling: cSpell:ignore tracestate
 weight: 1
 ---
 
-[OpenTelemetry for Ruby][repository] can be used to add automatic and manual
-instrumentation to your applications. Automatic instrumentation is enabled by
-adding [instrumentation packages][auto]. Manual instrumentation can be added
-using the [OpenTelemetry API][manual].
+This page will show you how to get started with OpenTelemetry in Ruby.
 
-### Requirements
+You will learn how you can instrument a simple application automatically, in
+such a way that [traces][], [metrics][] and [logs][] are emitted to the console.
 
-These instructions will explain how to set up automatic and manual
-instrumentation for a Ruby service. In order to follow along, you will need:
+## Prerequisites
+
+Ensure that you have the following installed locally:
 
 - MRI Ruby >= `3.0`, jruby >= `9.3.2.0`, or truffleruby >= 22.1
-- Docker Compose
+- [Bundler](https://bundler.io/)
+
+{{% alert color="warning" %}} jruby only targets compatibility with MRI Ruby
```

Comment:
The alert should have a title:

```suggestion
{{% alert  title="Warning" color="warning" %}} `jruby` only targets compatibility with MRI Ruby
```

If you don't like "Warning" you can choose another title.